modules/nodes-pods-autoscaling-custom-creating-workload.adoc

```diff
@@ -0,0 +1,158 @@
+Timestamp:             2019-05-23T18:47:56Z
+Window:                1m0s
+Events:                <none>
+----
+
+* The Custom Metrics Autoscaler Operator must be installed. 
+
+.Procedure
+
+. Create a YAML file similar to the following:
++
+.Example scaled object
+[source,yaml,options="nowrap"]
+----
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: scaledobject
+  namespace: my-namespace
+spec:
+  scaleTargetRef:
+    api: apps/v1 <1>
+    name: example-deployment <2>
+    kind: Deployment <3>
+    envSourceContainerName: .spec.template.spec.containers[0] <4>
+  cooldownPeriod:  200 <5>
+  maxReplicaCount: 100 <6>
+  minReplicaCount: 0 <7>
+  pollingInterval: 30 <8>
+  fallback: <9>
+    failureThreshold: 3
+    replicas: 6
+  advanced: 
+    restoreToOriginalReplicaCount: false <10>
+    horizontalPodAutoscalerConfig: 
+      behavior: <11>
+        scaleDown:
+          stabilizationWindowSeconds: 300
+          policies:
+          - type: Percent
+            value: 100
+            periodSeconds: 15
+  triggers:
+  - type: prometheus <12>
+    metadata:
+      serverAddress: https://thanos-querier.openshift-monitoring.svc.cluster.local:9092
+      namespace: kedatest 
+      metricName: http_requests_total
+      threshold: '5'
+      query: sum(rate(http_requests_total{job="test-app"}[1m]))
+      authModes: "basic"
+  - authenticationRef: <13>
+      name: prom-triggerauthentication
+    metadata:
+      name: prom-triggerauthentication
+    type: object
+  - authenticationRef: <14>
+      name: prom-cluster-triggerauthentication
+    metadata:
+      name: prom-cluster-triggerauthentication
+    type: object
+----
+<1> Optional: Specifies the API version of the target resource. The default is `apps/v1`.
+<2> Specifies the name of the object that you want to scale. 
+<3> Specifies the `kind` as `Deployment`, `StatefulSet` or `CustomResource`.
+<4> Optional: Specifies the name of the container in the target resource, from which the custom autoscaler gets environment variables holding secrets and so forth. The default is `.spec.template.spec.containers[0]`.
+<5> Optional. Specifies the period in seconds to wait after the last trigger is reported before scaling the deployment back to `0` if the `minReplicaCount` is set to `0`. The default is `300`.
+<6> Optional: Specifies the maximum number of replicas when scaling up. The default is `100`.
+<7> Optional: Specifies the minimum number of replicas when scaling down.
+<8> Optional: Specifies the interval in seconds to check each trigger on. The default is `30`.
+<9> Optional: Specifies the number of replicas to fall back to if a scaler fails to get metrics from the source for the number of times defined by the `failureThreshold` parameter. For more information on fallback behavior, see the https://keda.sh/docs/2.7/concepts/scaling-deployments/#scaledobject-spec[KEDA documentation]. 
```

Comment:
If we were to use a link it should be https://keda.sh/docs/2.7/concepts/scaling-deployments/#fallback not the one currently in use. 

That said; I think the information that the upstream has could be/should be down streamed (thus I don't understand the need for the link. 

This information is critical to understanding a key feature / capability of the offering; and should be included in the docs for the functionality (IMO). As a result I am not supportive of a link here.

Reply:
@gauravsingh85 @joelsmith PTAL at Eric's concerns.

Reply:
@mburke5678 fallback is in the part of APPENDIX, so I did not include it in the test scope. https://github.com/zroubalik/keda-openshift-examples/tree/main/kafka/amqstreams

Reply:
@weinliu I removed `fallback` from the docs